src/Utils.ts

```diff
@@ -18,7 +18,8 @@ function readRegistryValue(subKey: string): Promise<string> {
         // eslint-disable-next-line
         regedit.list(FS_KEY, (err: any, result: any) => {
             if (err) {
-                throw Error(`Failed to read registry value ${FS_KEY} (${err})`);
+                console.error(`Failed to read registry value ${FS_KEY} (${err})`);
+                resolve('2048');
```

Comment:
Just a suggestion for keeping the "responsibility" of the two functions clear:

```suggestion
                reject(`Failed to read registry value ${FS_KEY} (${err})`);
```
and handle the rejection in this function, which is responsible for finding the port number.
```ts
async function findSimConnectPortIPv4(): Promise<number> {
    try {
        const port = await readRegistryValue('SimConnect_Port_IPv4');
        return parseInt(port, 10);
    } catch {
        return 2048;
    }
}
```

Reply:
Fixed that ;)